src/multidim/intervalbox.jl

```diff
@@ -140,3 +140,18 @@ end
 
 hull(a::IntervalBox{N,T}, b::IntervalBox{N,T}) where {N,T} = IntervalBox(hull.(a[:], b[:]))
 hull(a::Vector{IntervalBox{N,T}}) where {N,T} = hull(a...)
+
+"""
+    zero_box(N, T)
+
+Return the zero interval box of dimension `N` in the numeric type `T`.
+"""
+zero_box(N, ::Type{T}) where T<:Real = IntervalBox(zero_interval(T), N)
```

Comment:
What about adding `zero(::Type{IntervalBox{N,T}}`?

Reply:
yes, we could add a new method such that eg. `zero(IntervalBox{2, Float64})` is the same as `zero_box(2, Float64)`. this is in fact open issue https://github.com/JuliaIntervals/IntervalArithmetic.jl/issues/438